awx_collection/plugins/doc_fragments/auth_plugin.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Wayne Witzel III <wayne@riotousliving.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    # Ansible Tower documentation fragment
+    DOCUMENTATION = r'''
+options:
+    host:
+        description: The network address of your Ansible Tower host.
+        env:
+            - name: TOWER_HOST
+    username:
+        description: The user that you plan to use to access inventories on Ansible Tower.
+        env:
+            - name: TOWER_USERNAME
+    password:
+        description: The password for your Ansible Tower user.
+        env:
+            - name: TOWER_PASSWORD
+    oauth_token:
+        description:
+            - The Tower OAuth token to use.
+        env:
+            - name: TOWER_OAUTH_TOKEN
+    verify_ssl:
+        description:
+            - Specify whether Ansible should verify the SSL certificate of Ansible Tower host.
+            - Defaults to True, but this is handled by the shared module_utils code
+        type: bool
+        env:
+            - name: TOWER_VERIFY_SSL
+        aliases: [ validate_certs ]
+
+notes:
+- If no I(config_file) is provided we will attempt to use the tower-cli library
+  defaults to find your Tower host information.
```

Comment:
> tower-cli library defaults

I'm trying to think if there's any reason for this bullet point to exist at all.... I'm not coming up with anything. I vote to delete it.

Should we also delete:

https://github.com/ansible/awx/blob/824d798d814cd63792c0a25a7ba4f55e9d16776e/awx_collection/plugins/doc_fragments/auth.py#L55-L56

?

Yes. Yes we should.

Reply:
Yea, this doc was almost a direct copy from the other one (as is also the case with the next comment). 

I feel like something should in here somewhere to indicate that we will, for backwards compatibility, search for this file like the old TowerCLI did. It would be nice to remove the TowerCLI reference, maybe we just have a small blurb with something like https://docs.ansible.com/ansible-tower/3.5.3/html/towerapi/tower_cli.html#file-locations?

Perhaps we let this go in as in as a clone and then cut a second PR to adjust this and the comment below?